.changeset/tough-candles-travel.md

```diff
@@ -0,0 +1,153 @@
+---
+'@makeswift/controls': patch
+'@makeswift/runtime': patch
+---
+
+### New `Font` control
+
+We now have a `Font` control. This control let's you select a `fontFamily`, `fontStyle`, and `fontWeight`.
+The values available are sourced from our Google Fonts integration within Makeswift and from the variants you pass to `getFonts` in your `[...makeswift]` route handler.
+
+Available params for the Font control include:
+
+- `label?: string`
+  - Text for the panel label in the Makeswift builder.
+- `variant?: boolean = true`
+  - Config for whether `fontStyle` and `fontWeight` are included in the final value. Defaults to true.
+    This value changes what panel inputs are shown in the Makeswift builder, and changes the type of `defaultValue`.
+- `defaultValue?: variant extends false ? {fontFamily: string} : { fontFamily: string, fontStyle: 'normal' | 'italic', fontWeight: 100 | 200 | 300 | 400 | 500 | 600 | 700 | 800 | 900 }`
+  - Config what the default option for this control is. Without `defaultValue` the resulting value passed to your component is optional.
```

Comment:
```suggestion
  - Configure what the default option for this control is. Without `defaultValue` the resulting value passed to your component is optional.
```

Reply:
What are your thoughts on: "- Config for the a default value passed to your component. Passing a value means the resulting data type is no longer optional?"

Reply:
I don't like using "config" as a verb. I usually only see "config" used to replace "configuration", not "configure" so changing it to be a noun like you have above is alright, but in that sense, this is a "config value", the "config" is the entire object itself. You could, like the previous one, say "The default value passed to your component when no value is available. Without `defaultValue` the..."